project_euler/validate_solutions.py

```diff
@@ -0,0 +1,783 @@
+#!/usr/bin/env python3
+import importlib.util
+import pathlib
+from types import ModuleType
+from typing import Generator, Tuple
+
+import pytest
+
+PROJECT_EULER_PATH = pathlib.Path.cwd().joinpath("project_euler")
+
+# https://github.com/luckytoilet/projecteuler-solutions/blob/master/Solutions.md
+ANSWERS: Tuple[Tuple[int, str], ...] = (
```

Comment:
Please put this data into a `euler_answers.json` file and then read in that file instead of putting all this content inside this Python file.